src/kaleidoscope/layers.cpp

```diff
@@ -46,6 +46,17 @@ Key Layer_::live_composite_keymap_[Runtime.device().numKeys()];
 uint8_t Layer_::active_layer_keymap_[Runtime.device().numKeys()];
 Layer_::GetKeyFunction Layer_::getKey = &Layer_::getKeyFromPROGMEM;
 
+void Layer_::setup() {
+  // Explicitly set layer 0's state to 1
+  bitSet(layer_state_, 0);
```

Comment:
We _could_ just initialize `layer_state_` above to the correct magic number, but... that'd be a magic number. It feels cleaner to initialize in `setup()`. But if requested, I'll update the initialization instead, and drop the `bitSet` from here.

(I'd still keep `setup()` regardless).